.github/workflows/build.yml

```diff
@@ -3,7 +3,7 @@ defaults:
   run:
     shell: bash
 env:
-  DOCKER_REGISTRY: hazelops
```

Comment:
We should keep the  `hazelops` registry. If it didn't work, it might be due to permissions.